server/controllers/cellMove/cellMoveTemporaryMove.ts

```diff
@@ -0,0 +1,56 @@
+import { Request, Response } from 'express'
+import { putLastNameFirst, formatLocation, formatName } from '../../utils'
+import PrisonerCellAllocationService from '../../services/prisonerCellAllocationService'
+
+type Params = {
+  prisonerCellAllocationService: PrisonerCellAllocationService
+}
+
+export default ({ prisonerCellAllocationService }: Params) =>
+  async (req: Request, res: Response) => {
+    const {
+      systemClientToken,
+      user: { activeCaseLoad },
```

Comment:
not wrong but not so sure about this way of renaming destructured variables ... 
perhaps on Line 30 we could just do user.activeCaseload etc